pkg/utils/fake_driver_utils.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+var _ StatsUtils = (*FakeStatsUtilsFuncStructImpl)(nil)
+
+type FakeStatsUtilsFuncStruct struct {
+	FuseMountFn      func(path string, comm string, args []string) error
+	FSInfoFn         func(path string) (int64, int64, int64, int64, int64, int64, error)
+	CheckMountFn     func(targetPath string) (bool, error)
+	FuseUnmountFn    func(path string) error
+	BucketToDeleteFn func(volumeID string) (string, error)
+}
```

Comment:
Remove FuseMountFn and FuseUnmountFn as these are not part of StatsUtils
And add missing method `GetBucketUsage`

Parallelly add GetBucketUsage fake method below